dev/PushNotifications/PushNotificationManager.cpp

```diff
@@ -157,8 +151,9 @@ namespace winrt::Microsoft::Windows::PushNotifications::implementation
                 {
                     if (IsActivatorSupported(PushNotificationRegistrationActivators::PushTrigger))
                     {
+                        auto appUserModelId{ PushNotificationHelpers::GetAppUserModelId() };
                         ChannelDetails channelInfo{};
-                        winrt::hresult hr = CreateChannelWithRemoteIdHelper(remoteId, channelInfo);
+                        winrt::hresult hr{ CreateChannelWithRemoteIdHelper(appUserModelId, remoteId, channelInfo) };
```

Comment:
Is the else condition even needed below? We always expect this API call to succeed right?

Reply:
I would double check if the E_NOTIMPL case is valid anymore and would delete the code in the else block if not.